docs/en/sql-reference/functions/uuid-functions.md

```diff
@@ -644,9 +640,7 @@ UUIDv7ToDateTime(uuid[, timezone])
 
 **Returned value**
 
-- Timestamp with milliseconds precision. If the UUID is not a valid version 7 UUID, it returns 1970-01-01 00:00:00.000.
-
-Type: [DateTime64(3)](/docs/en/sql-reference/data-types/datetime64.md).
+- Timestamp with milliseconds precision. If the UUID is not a valid version 7 UUID, it returns 1970-01-01 00:00:00.000. [DateTime64(3)](/docs/en/sql-reference/data-types/datetime64.md).
```

Comment:
(It is a bit inconsistent that we sometimes reference to data type docs (e.g. here: `DateTime64`)
- using `../../` as in url-functions.md, l. 265
- using `../` as in this file, l. 683,
- using an absolute link like here (l. 643).

Would it be possible to fix that? I think that the convention is to use the second form. Totally okay, to push this as a follow-up PR.)